prometheus_es_exporter/__init__.py

```diff
@@ -344,9 +348,9 @@ def main():
     port = args.port
     es_cluster = args.es_cluster.split(',')
     if args.ca_certs:
-        es_client = Elasticsearch(es_cluster, verify_certs=True, ca_certs=args.ca_certs)
+        es_client = Elasticsearch(es_cluster, verify_certs=True, ca_certs=args.ca_certs, http_auth=(args.user, args.password))
     else:
-        es_client = Elasticsearch(es_cluster, verify_certs=False)
+        es_client = Elasticsearch(es_cluster, verify_certs=False, http_auth=(args.user, args.password))
```

Comment:
I suspect this won't work as expected when the user and password aren't provided.

Instead, please construct a `http_auth` variable earlier, and pass it in here. Set it to `None`, instead of a tuple, if neither user or password are provided.

Will also need to handle one being specified, and not the other. e.g. if only user is specified, use `parser.error()` to tell the user that `--basic-user` requires `--basic-password`, and vice versa.